parcels/field.py

```diff
@@ -1880,9 +1898,11 @@ def _get_initial_chunk_dictionary(self):
                 init_chunk_dict[depthname] = max(1, depthvalue)
         # ==== closing check-opened requested dataset ==== #
         self.dataset.close()
-        # ==== check if the chunksize reading is successfull. if not, load the file ONCE really into memory and ==== #
+        # ==== check if the chunksize reading is successful. if not, load the file ONCE really into memory and ==== #
         # ==== deduce the chunking from the array dims.                                                         ==== #
         try:
+            if len(init_chunk_dict) < 3:
```

Comment:
Why is this the case? What if a field only has two dimensions (e.g. bathymetry)?

Reply:
if a field only has 2 dimensions by design, we don't do chunking anyway. So if parcels requests chunking and is not able to detect 3 or more working chunking dimensions, something has gone wrong.